src/containers/Tenant/Diagnostics/Partitions/PartitionsControls/PartitionsControls.tsx

```diff
@@ -137,25 +132,24 @@ export const PartitionsControls = ({
         onHiddenColumnsChange(result);
     };
 
+    const renderOption = (option: SelectOption) => {
+        return (
+            <div className={b('select-option', {empty: option.value === ''})}>{option.content}</div>
+        );
+    };
+
     return (
         <div className={b('controls')}>
             <Select
                 className={b('consumer-select')}
-                placeholder={i18n('controls.consumerSelector.placeholder')}
```

Comment:
We don't need placeholder, as we now always have empty option ({value: "", content: "No consumer"})